Cargo.toml

```diff
@@ -8,6 +8,8 @@ debug = 0
 
 [profile.release]
 incremental = true
+lto = true
```

Comment:
Since `cargo` now has support for [custom profiles](https://doc.rust-lang.org/stable/cargo/reference/manifest.html#the-profile-sections), could we make a `retail` profile that's used for the binaries we ship?

Reply:
Ah I'm guessing you currently use release mode also in your dev cycle. 
Ok a retail mode probably is a good call then.